CHANGELOG.md

```diff
@@ -104,6 +104,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Compress and cache cluster state during validate join request ([#7321](https://github.com/opensearch-project/OpenSearch/pull/7321))
 - [Snapshot Interop] Add Changes in Create Snapshot Flow for remote store interoperability. ([#7118](https://github.com/opensearch-project/OpenSearch/pull/7118))
 - Add new query profile collector fields with concurrent search execution ([#7898](https://github.com/opensearch-project/OpenSearch/pull/7898))
+- Update version guards to deal for new query profile collector fields ([#7979](https://github.com/opensearch-project/OpenSearch/pull/7979))
```

Comment:
We don't need the changelog entry (this is essentially the part of the `Add new query profile collector fields with concurrent search execution`), thank you

Reply:
Got it. It has been addressed.